app/build.gradle

```diff
@@ -1,39 +1,41 @@
-apply(plugin: "com.android.application")
-apply(plugin: "kotlin-android")
-apply(plugin: "kotlin-android-extensions")
-apply(plugin: "kotlin-kapt")
+apply plugin: "com.android.application"
+apply plugin: "kotlin-android"
+apply plugin: "kotlin-android-extensions"
+apply plugin: "kotlin-kapt"
+
+apply from: '../environments.gradle'
 
 android {
-    compileSdkVersion(29)
-    buildToolsVersion("29.0.3")
+    compileSdkVersion rootProject.ext.androidCompileSdkVersion
+
     defaultConfig {
         applicationId = "org.covital"
-        minSdkVersion(23)
-        targetSdkVersion(29)
-        versionCode(1)
-        versionName("1.0")
-        testInstrumentationRunner("androidx.test.runner.AndroidJUnitRunner")
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        versionCode rootProject.ext.versionCode
+        versionName rootProject.ext.versionName
+        testInstrumentationRunner rootProject.ext.testInstrumentationRunner
     }
     buildTypes {
         debug {
             minifyEnabled = false
-            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+            proguardFiles getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro"
         }
         release {
             minifyEnabled = true
-            proguardFiles(getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro")
+            proguardFiles getDefaultProguardFile("proguard-android-optimize.txt"), "proguard-rules.pro"
         }
     }
 
     flavorDimensions "default"
     productFlavors {
         med {
             dimension "default"
-            buildConfigField "String", "BASE_URL", "https://guarded-crag-28391.herokuapp.com/api-docs/#/default/"
+            buildConfigField "String", "BASE_URL", "\"" + environments.med.baseURL + "\""
```

Comment:
This is fine for now but why don't we just do this via Koin? It'll be easier later if we have an internal build version that wants to switch between different hostnames + API keys for different environments.